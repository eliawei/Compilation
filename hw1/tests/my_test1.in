umm // \n/nl;kopwqke0912md0"''\0\x11b
b8byte8if08;;b 8byte 8if 08;;//
"" "\x7A" "\\" "\raa\n\rbb\n\0cc" "//" "\t\"\""
"\q