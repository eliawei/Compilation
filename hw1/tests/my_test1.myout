1 ID umm
1 COMMENT //
2 ID b8byte8if08
2 SC ;
2 SC ;
2 B b
2 NUM 8
2 BYTE byte
2 NUM 8
2 IF if
2 NUM 0
2 NUM 8
2 SC ;
2 SC ;
2 COMMENT //
Error "
3 STRING  
Error \
3 ID x7A
3 STRING  
Error \
Error \
3 STRING  
Error \
3 ID raa
Error \
3 ID n
Error \
3 ID rbb
Error \
3 ID n
Error \
3 NUM 0
3 ID cc
3 STRING  
3 COMMENT //
Error "
Error \
4 ID q
