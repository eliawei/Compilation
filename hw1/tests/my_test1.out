1 ID umm
1 COMMENT //
2 ID b8byte8if08
2 SC ;
2 SC ;
2 B b
2 NUM 8
2 BYTE byte
2 NUM 8
2 IF if
2 NUM 0
2 NUM 8
2 SC ;
2 SC ;
2 COMMENT //
3 STRING 
3 STRING z
3 STRING \
3 STRING aa
bb

3 STRING //
3 STRING 	""
Error unclosed string
